package.json

```diff
@@ -108,7 +105,6 @@
     "karma-webpack": "1.7.0",
     "mocha": "2.3.4",
     "moment": "2.12.0",
-    "react-test-context": "0.1.0",
```

Comment:
I think we were using it in the test framework before there were some react-intl updates.